x-pack/legacy/plugins/apm/public/components/app/Settings/ApmIndices/index.tsx

```diff
@@ -22,6 +22,7 @@ import {
 import { useFetcher } from '../../../../hooks/useFetcher';
 import { useCallApmApi } from '../../../../hooks/useCallApmApi';
 import { APMClient } from '../../../../services/rest/createCallApmApi';
+import { clearCache } from '../../../../services/rest/callApi.ts';
```

Comment:
The type check error is here:

> An import path cannot end with a '.ts' extension. Consider importing '../../../../services/rest/callApi' instead.

```suggestion
import { clearCache } from '../../../../services/rest/callApi';
```

Reply:
My bad! Fixed